x-pack/plugins/actions/server/lib/action_executor.ts

```diff
@@ -112,7 +112,7 @@ export class ActionExecutor {
     const actionLabel = `${actionTypeId}:${actionId}: ${name}`;
     const event: IEvent = {
       event: { action: EVENT_LOG_ACTIONS.execute },
-      kibana: { namespace, saved_objects: [{ type: 'action', id: actionId }] },
+      kibana: { saved_objects: [{ type: 'action', id: actionId, namespace }] },
```

Comment:
should probably fix this issue, while you're here: https://github.com/elastic/kibana/issues/63134 - basically, we're writing the "space_id" value here, not the "namespace" value.